src/app/app-design-tool/components/app-form-design/app-form-design.component.ts

```diff
@@ -181,4 +185,8 @@ export class AppFormDesignComponent implements OnInit, AfterViewInit {
       project_id: '',
     });
   }
+
+  formSubmitted(): void {
+    this.buttonClicked = !this.buttonClicked;
```

Comment:
just include this line in `onSubmit()`, and remove this function.

Reply:
Done!